tests/ctl/core/test_api.py

```diff
@@ -1715,7 +1715,7 @@ def test_system_update_updates_declarations(
         "update_declarations",
         [
             (
-                [  # update a dec matching one existing dec
+                [  # Check 1: update a dec matching one existing dec
```

Comment:
nit/just a thought - the pytest output actually starts the parameterization at `0`, it _could_ be a bit easier to line this up with testing output if we also started our indexing at `0` in the comments here? it just tripped me up for a quick second while i was debugging one of the failing parameterizations 😄

Reply:
rookie mistake on my part! You're totally right

We could also pass an `id` to pytest to make the parametrized run more clear

Reply:
oooh cool, that's a nice protip...